### ISSUE TITLE
SEGV (with newest Clone.pm?)

### ISSUE THREAD

**Issue by eserte** [pombase/chobo#12]
t/ontology_term.t fails on some of my smokers with a segmentation fault:
```
t/ontology_term.t .......... 
Failed 9/15 subtests 
...
t/ontology_term.t        (Wstat: 11 Tests: 6 Failed: 0)
  Non-zero wait status: 11
  Parse errors: Bad plan.  You planned 15 tests but ran 6.
```

Statistical analysis suggests that this happens with Clone 0.40 (@garu: FYI):
```
****************************************************************
Regression 'mod:Clone'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	28514808701296756.00
[1='eq_0.39']  	     -0.0000	      0.0000	  -5.90
[2='eq_0.40']  	     -1.0000	      0.0000	-26749261624182508.00

R^2= 1.000, N= 45, K= 3
****************************************************************
```

**Comment by kimrutherford** [comment]
Might be because of: garu/Clone#10

**Comment by garu** [comment]
@eserte @kimrutherford I'm sorry this caused issues. Clone 0.41 is already on CPAN. Please update and let me know if the related test failure still happens!

**Comment by kimrutherford** [comment]
Thank you both for your help.  The tests run for me locally so I've released a new version that requires Clone v0.41.
